AlwaysAllowChargingSounds/build.gradle.kts

```diff
@@ -0,0 +1,15 @@
+plugins {
+    alias(libs.plugins.buildlogic.android.application)
+    alias(libs.plugins.buildlogic.kotlin.android)
+}
+
+android {
+    namespace = "de.binarynoise.AlwaysAllowChargingSounds"
+    
+    defaultConfig {
+        minSdk = 28
+        targetSdk = 36
```

Comment:
Targeting `36` but only tested on `35`. Implementation seems not to change though, looking at the source.